### ISSUE TITLE
Tests failing when running on 'lc_numeric', 'lc_time' equal to "Polish_Poland.1250"

### ISSUE THREAD

**Issue by jszczepankiewicz** [impossibl/pgjdbc-ng#291]
Hi, I'm trying to run locally tests. I have Windows 10 running on Polish / Poland locale. Query: 
`SELECT name, setting FROM pg_settings WHERE name IN ('lc_numeric', 'lc_time')`
returns:
"lc_numeric";"Polish_Poland.1250"
"lc_time";"Polish_Poland.1250"

tests are failing with:
```
java.util.IllformedLocaleException: Ill-formed region: Poland [at index 0]
	at java.util.Locale$Builder.setRegion(Locale.java:2495)
	at com.impossibl.postgres.system.BasicContext.loadLocale(BasicContext.java:285)
	at com.impossibl.postgres.system.BasicContext.init(BasicContext.java:265)
	at com.impossibl.postgres.jdbc.PGConnectionImpl.init(PGConnectionImpl.java:249)
	at com.impossibl.postgres.jdbc.ConnectionUtil.createConnection(ConnectionUtil.java:182)
	at com.impossibl.postgres.jdbc.PGDriver.connect(PGDriver.java:77)
	at com.impossibl.postgres.jdbc.PGDriver.connect(PGDriver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:208)
	at com.impossibl.postgres.jdbc.TestUtil.openDB(TestUtil.java:128)
	at com.impossibl.postgres.jdbc.TestUtil.openDB(TestUtil.java:116)
	at com.impossibl.postgres.jdbc.ExceptionTest.setUp(ExceptionTest.java:48)
```

Will try to find solution but as I'm newby in this code maybe somebody may give me some high level instruction how to fix that (hopefully without forcing postgresql to some more popular local.
Thanks
JS

**Comment by jszczepankiewicz** [comment]
I solved the problem running tests by modification:

```
switch (localeSpec.toUpperCase(Locale.US)) {
        case "C":
        case "POSIX":
          localeSpec = "en_US";
          break;        
      }
```

changing to:

```
switch (localeSpec.toUpperCase(Locale.US)) {
        case "C":
        case "POSIX":
          localeSpec = "en_US";
          break;
        case "POLISH_POLAND.1250":
          localeSpec = "pl_PL";
          break;
      }
```

now some tests related to currency are failing but most of them are passing:

```
Failed tests: 
  CodecTest.testTextCodecs:156->test:264->test:294 expected:<2342.00> but was:<2342>
  CodecTest.testTextCodecs:156->test:294 expected:<2342.00> but was:<2342>
  CodecTest.testTextCodecs:156->test:277->test:294 expected:<2342.00> but was:<2342>
Tests in error: 
  CodecTest.testTextCodecs:156->inOut:228 » NumberFormat For input string: "0,00...
  CodecTest.testTextCodecs:156->inOut:228 » NumberFormat For input string: "0,00...
  CodecTest.testTextCodecs:156->inOut:228 » NumberFormat For input string: "0,00...

Tests run: 774, Failures: 3, Errors: 3, Skipped: 21
```

**Comment by jesperpedersen** [comment]
Should be fixed by #287
